spec/podfile/target_definition_spec.rb

```diff
@@ -294,6 +294,7 @@ module Pod
         @parent.use_frameworks!
         @child.use_frameworks!(false)
         @child.should.not.uses_frameworks?
+        @child.should.not.uses_frameworks?
```

Comment:
Please add a comment saying why this line is necessary, since it looks superfluous.